src/main/java/hudson/plugins/promoted_builds/jobdsl/ConditionsContext.java

```diff
@@ -0,0 +1,157 @@
+package hudson.plugins.promoted_builds.jobdsl;
```

Comment:
maybe promoted_builds.integrations.jobdsl
There will be definitely an entry for Pipeline Plugin (fka Workflow) at some point